app/views/reporting/base/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title t('.page_title') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">
+      <%= t '.heading' %>
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <% if current_user.reports.include?('caseworker_report') %>
+      <h3 class='govuk-heading-m'>
+        <% report = Reporting::MonthlyCaseworkerReport.latest %>
```

Comment:
Could this be in the controller, and then passed as a variable?

Reply:
Yes, this page is just a convenience at the moment. How users navigate to reports is still under consideration.